master/docs/guides/svn.rst

```diff
@@ -0,0 +1,92 @@
+=================================
+Subversion Version Control System
```

Comment:
Maybe split each of the sections here into a separate guide?  Those after the first could refer back to the first ("You should be familiar with..").  So the guide heading would be "Building Every Commit to Trunk with Subversion"